index.js

```diff
@@ -1,29 +1,53 @@
 'use strict';
+var regexps = require('./regexps.json');
+const dayMilliSeconds = 24 * 60 * 60 * 1000;
+// A list of strings along with the offset to the date that they create
+const specialStrings = {
+  'yesterday': -1 * dayMilliSeconds,
+  'tomorrow': +1 * dayMilliSeconds,
+  'day before yesterday': -2 * dayMilliSeconds,
+  'day after tomorrow': +2 * dayMilliSeconds
+};
+const periods = {
+  'day': dayMilliSeconds,
+  'week': 7 * dayMilliSeconds
+};
+const days = { 'Sun': 0, 'Mon': 1, 'Tues': 2, 'Tue': 2, 'Wednes': 3, 'Wed': 3, 'Thurs': 4, 'Fri': 5, 'Sat': 6, 'Satur': 6 };
+
+const months = {'January': 0, 'Jan': 0, 'February': 1, 'Feb': 1, 'March': 2, 'Mar': 2, 'April': 3, 'Apr': 3, 'May': 4,
```

Comment:
Okay, refer to this: https://github.com/icyflame/relative-date-reverse/pull/3#issuecomment-228966604

Instead of two entries for `Jan` and `January`, let's just have one entry for `Jan`, and always strip the string that we need to match to this variable to the first three letters.